README.md

```diff
@@ -21,7 +21,10 @@ use Supabase\Storage;
 
 $api_key = getenv('API_KEY');
 $reference_id = getenv('REFERENCE_ID');
-$client = new  StorageClient($api_key, $reference_id);
+$scheme = 'http';
```

Comment:
After the __constructor is updated none of these should be needed - please remove them.